### DISCUSSION TITLE
Error cmp.entry.get_documentation overwritten

### DISCUSSION THREAD

**Discussion by xtheeq** [LazyVim/LazyVim#4132] [Q&A] [answered]
![Screenshot 2024-07-21 210706](https://github.com/user-attachments/assets/45f9bc25-193b-4033-8b53-b197903aebde)

**Comment by dpetka2001** [comment]
Keep an eye on https://github.com/folke/noice.nvim/issues/910 and https://github.com/folke/lazy.nvim/commit/18d1c1b47e175cd58dc12bf4792ef4e9a50505fa, which seem to be related. Hopefully a fix will be available soon.

**Comment by folke** [comment, marked as answer]
This should no longer happen with the latest lazy.nvim and noice.nvim.
If the problem persists, let me know.

**Reply by xtheeq** [reply]
Thanks, but now I'm facing another error
![Screenshot 2024-07-22 173954](https://github.com/user-attachments/assets/879fbac2-f4f8-4047-bfab-beeaea4076a2)
![Screenshot 2024-07-22 174023](https://github.com/user-attachments/assets/e6b3022f-f4a4-48c8-b029-395e3d59127c)

**Reply by folke** [reply]
That has nothing to do with noice nor LAzyVim.
Check the treesitter docs

**Reply by xtheeq** [reply]
![image](https://github.com/user-attachments/assets/984544af-0858-4b4a-a66b-ea83e409cc69)
Why doesn't it recognize the git?

**Reply by folke** [reply]
Your shell setup is messed up. It appears that `git --version` returns an error in Neovim, while it still gives the version.

**Reply by xtheeq** [reply]
it is,
clink was the reason
https://github.com/ScoopInstaller/Scoop/issues/1807#issuecomment-342422385
solved by uninstalling 
thanks folke the man the myth the legend..

**Reply by xtheeq** [reply]
![image](https://github.com/user-attachments/assets/84d61afb-e127-4061-9485-efc7f512cf22)
the issue persists,
I'm in the latest update btw

**Reply by ibeex** [reply]
Same her all on latest and I still get this error. Is there any info I can provide?

**Reply by rogeriofrsouza** [reply]
I have the same error here.

**Reply by folke** [reply]
I just changed something, but not sure that will fix it.
I don't get the error.

**Reply by Kayzels** [reply]
It happens for me as well, but it seems to happen only when opening Telescope from Dashboard using one of the Dashboard shortcuts. Opening a file directly in Neovim, or using the Telescope command directly doesn't do it, but typing the shortcut `f`, for example, to find files, leads to this.

It also only happens if I try do it too quickly, so possibly before all plugins have loaded. If I open Neovim, and wait a few seconds before doing anything, it doesn't show up. I cannot reproduce with default LazyVim, because it loads too quickly. I have one or two plugins that slow down Neovim, that actually allow me to see this.

**Reply by harshv5094** [reply]
It happens to me also, but when I use Telescope and also when I shift from dashboard buffer to file buffer
